<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QDialog" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1312</width>
    <height>715</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="barre_nav">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>1221</width>
     <height>701</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#barre_nav {
	background-color: rgb(0, 85, 127);
    background-color: #0F2434;
    border-radius: 8px;
    padding: 5px;
}

#candidat_btn,
#vehicule_btn,
#moniteur_btn,
#seance_btn,
#emp_btn,
#fermer_btn {
    color: #FBFAF5;
    background-color: transparent;
    border: none;
    padding: 8px 15px;
    font-weight: bold;
}

#candidat_btn:hover,
#vehicule_btn:hover,
#moniteur_btn:hover,
#seance_btn:hover,
#emp_btn:hover,
#fermer_btn:hover {
    color: #1D9E3A;
}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>271</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="ressources.qrc">:/yyyy/images/logo.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>220</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Nom d'utilisateur:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>320</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Mot de pass:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_login">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>220</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_password">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>320</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::EchoMode::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="motdepassoubli_E">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>390</y>
      <width>151</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(15, 36, 52);
color: rgb(29, 158, 58);</string>
    </property>
    <property name="text">
     <string>Mot de pass oublié?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="seconnecter_E">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>450</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#seconnecter_E {
    background-color: rgb(29, 158, 58); /* couleur normale */
    color: white;                       /* couleur du texte */
    border: none;
    padding: 5px 10px;
    border-radius: 5px;
}

#seconnecter_E:pressed {
    background-color: #239539; /* couleur quand le bouton est cliqué */
}
</string>
    </property>
    <property name="text">
     <string>Se connecter</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
